src/Controllers/PushController.js

```diff
@@ -38,7 +38,7 @@ export class PushController extends AdaptableController {
     return !!this.adapter;
   }
 
-  sendPush(body = {}, where = {}, config, auth, wait) {
+  sendPush(body = {}, where = {}, config, auth, pushStatusIdCallback = () => {}) {
```

Comment:
I understand that's not super nice, but I didn't want to break the returned promise that allow us to test the push flow